src/main/java/seedu/mindmymoney/helper/AddCommandInputTests.java

```diff
@@ -114,4 +130,22 @@ public static void testAmount(String inputAmount) throws MindMyMoneyException {
         }
         assert inputAmountAsInteger > 0 : "Amount should have a positive value";
     }
+
+    public static void testIncomeAmount(int inputAmount) throws MindMyMoneyException {
+        if (inputAmount < 0) {
+            throw new MindMyMoneyException("Amount cannot be negative!");
+        } else if (inputAmount > 1000000) {  // Students typically would not have an income greater than 1,000,000
```

Comment:
If i remember correctly, i think the comment should be above instead of beside the code.

Reply:
Ya good point, but in this case how to put above, cos it will intefere with the if else structure...